### PR TITLE
cantarell_fonts: 0.0.25 → 0.100

### PR DESCRIPTION
**Pull request by jtojnar**
###### Motivation for this change
Update & add `updateScript`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @fuuzetsu

### CODE REVIEW

**Review by Fuuzetsu — APPROVED** [review]

**Comment by Fuuzetsu** [comment]
@GrahamcOfBorg build cantarell-fonts

**Comment by Fuuzetsu** [comment]
hm, let me try again

@GrahamcOfBorg build cantarell-fonts-0.100

**Comment by jtojnar** [comment]
I am not sure if it should still be a fixed output derivation.

**Comment by jtojnar** [comment]
Also there no longer seems to be `$out/share/fontconfig/conf.avail/31-cantarell.conf`.
